db/utility.js

```diff
@@ -21,9 +22,15 @@ const handleComment = (itemTransResult, itemCommentResult, commentResult)=>{
             }
         })
     })
+    // console.log("itemCommentObj", itemCommentObj)
     return itemCommentObj
 }
-
+const makeImgUrl = (imgFromDb)=>{
+    // todo is there a way to get current backend url?
```

Comment:
did you check stack overflow? https://stackoverflow.com/questions/10183291/how-to-get-the-full-url-in-express

Reply:
I searched it and saw this post. But, we upload image via GraphQL instead of express. Therefore, I think it did work.